content/publication_resources/bibtex/lahti.bib

```diff
@@ -1,4 +1,15 @@
 
+@Article{Khalighi2024acm,
+  author =	 {Moein Khalighi and Giulio Benedetti and Leo Lahti},
+  title =	 {Algorithm XXXX: FdeSolver, a Julia Package for Solving Fractional Differential Equations},
```

Comment:
Add double curly brackets here like {{}} because that will save the intentional capitalization for certain words.

I think the "Algorithm XXXX: " could be removed, it is some extra string from the journal.

Reply:
corrected